var/spack/repos/builtin/packages/ImageMagick/package.py

```diff
@@ -26,38 +26,25 @@
 
 
 class Imagemagick(Package):
-    """ImageMagick is a image processing library"""
-    homepage = "http://www.imagemagic.org"
+    """ImageMagick is a software suite to create, edit, compose,
+    or convert bitmap images."""
 
-    # -------------------------------------------------------------------------
-    # ImageMagick does not keep around anything but *-10 versions, so
-    # this URL may change.  If you want the bleeding edge, you can
-    # uncomment it and see if it works but you may need to try to
-    # fetch a newer version (-6, -7, -8, -9, etc.) or you can stick
-    # wtih the older, stable, archived -10 versions below.
-    #
-    # TODO: would be nice if spack had a way to recommend avoiding a
-    # TODO: bleeding edge version, but not comment it out.
-    # -------------------------------------------------------------------------
-    # version('6.9.0-6', 'c1bce7396c22995b8bdb56b7797b4a1b',
-    # url="http://www.imagemagick.org/download/ImageMagick-6.9.0-6.tar.bz2")
+    homepage = "http://www.imagemagick.org"
+    url      = "http://www.imagemagick.org/download/ImageMagick-7.0.2-6.tar.gz"
```

Comment:
Contrary to the comments I removed, I don't think the "stable" -10 releases are kept around either. I don't see them available for download. They only have the bleeding edge of the current release (7.0.2-7) and the previous release (6.9.5-5). There was a [recent security vulnerability discovered](https://imagetragick.com/), so that may also be why. I'm going to use the GitHub repo since that seems to be the only one with old releases. GitLab also has them I think